include/clap/ext/params.h

```diff
@@ -84,40 +84,21 @@ enum {
    // if so the double value is converted to integer using a cast (equivalent to trunc).
    CLAP_PARAM_IS_STEPPED = 1 << 0,
 
-   // Does this param supports per note automations?
-   CLAP_PARAM_IS_PER_NOTE = 1 << 1,
-
-   // Does this param supports per channel automations?
-   CLAP_PARAM_IS_PER_CHANNEL = 1 << 2,
-
-   // Does this param supports per port automations?
-   CLAP_PARAM_IS_PER_PORT = 1 << 3,
-
    // Useful for for periodic parameters like a phase
```

Comment:
`for for`. Also, might be worth documenting how this works. Is this the same as VST3's wraparound thing? (where values outside of `[0, 1]` should be treated as ``x `fmod` 1.0``? This is a documentation thing though, so it's probably better to go through all of this once before 1.0 since there a couple other things that could be documented better.

Reply:
Yes, it is like fmod: `((value - min) % (max - min)) + min`